benchmarks/pubmed_sentence_classifier/convert_to_classifier_input.py

```diff
@@ -72,30 +85,39 @@ def parse_raw_file(raw_file_path:Path)->Iterable[Record]:
   preloader.register(*embedding_util.get_scibert_initializer(
       scibert_data_dir=args.bert_data_dir,
   ))
-  dpg.add_global_preloader(client=dask_client, preloader=preloader)
-
-  print("Converting to records.")
-  records = parse_raw_file(args.raw_data_in)
-
-  # Step 3: Embed Records
-  embedded_records = embedding_util.embed_records(
-      records,
-      batch_size=args.batch_size,
-      text_field="text",
-      max_sequence_length=args.max_sequence_length,
-  )
-
-  # Step 4: Records to training data via util
-  print("Converting to sentence_classifier.util.TrainingData")
-  training_data = [
-      sent_class_util.record_to_training_tuple(r)
-      for r in embedded_records
-  ]
-
-  print("Saving as mock ckpt")
-  done_file = args.eval_data_out.joinpath("__done__")
-  part_file = args.eval_data_out.joinpath("part-0.pkl")
-  with open(part_file, 'wb') as f:
-    pickle.dump(training_data, f)
-  with open(done_file, 'w') as f:
-    f.write(f"{part_file}\n")
+  dpg.add_global_preloader(preloader=preloader)
```

Comment:
Changes to the global preloader allow me to get away with not specifying a client.